website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,102 @@
+## Permissions 
+
+When a user does not have permissions, hide the related actions, navigation items and views.
+
+### Hide/show relevant UI
+
+In this example, the user's role is **viewer**, and they cannot deploy or destroy a cluster. An explanation is unnecessary as users do not require additional functionality for their day-to-day tasks.
+
+![Image showing two dropdown with lists open, one with only one menu item that is reader specific, while the other has more options that allow for further capabilities.](/assets/patterns/disabled-patterns/permissions-example.png)
+
+Similarly, if a user has the permissions to update their software, show the relevant UI for them to do just that.
+
+
+### Provide guidance
+
+When a user lands on a page they can't access, indicate why it's unavailable and provide guidance so the user can fix the error. Consider displaying a call to action so the user doesn't feel stuck.
+
+![Showing a 403 error page with information about what is going on, with a way to go back and a way to log in with a different token](/assets/patterns/disabled-patterns/permissions-example-3.png)
+
+## Conditional availability
+
+When other steps must be taken to enable a feature or function, provide context in place of that feature so users can enable this feature.
```

Comment:
[suggestion] I think this could be streamlined a bit so you aren't using "feature" so many times.

```suggestion
When other steps must be taken to enable a feature or function, provide context in place of that feature so users can enable it.
```